osbs/utils.py

```diff
@@ -289,6 +289,10 @@ def get_time_from_rfc3339(rfc3339):
         return timegm(time_tuple)
 
 
+
+VALID_BUILD_CONFIG_NAME_CHARS = re.compile('[a-z0-9]')
```

Comment:
We need to strip both `-` and `.` from both ends of `repo` and `branch`. But this doesn't make me feel very confident that the issue is fixed for good....